README.md

```diff
@@ -16,6 +16,9 @@ An interactive terminal interface for [Zulip](https://zulipchat.com).
 - OSX
 - WSL (On Windows)
 
+## What do you see
```

Comment:
For the `README`, you might want to use a heading that's more appropriate than 'What do you see'.

Reply:
Sure, I will change the heading.

Reply:
I wasn't able to understand what @neiljp mentioned regarding the announce stream. I will be glad if you can elaborate a little bit.

Reply:
@aditya-sarkar441 It seems that you resolved the #announce issue, based on the last screenshot.